<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FleetDialog</class>
 <widget class="QDialog" name="FleetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>322</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>814</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>814</width>
    <height>322</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fleet Roster Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../nexus.qrc">
    <normaloff>:/images/favicon.png</normaloff>:/images/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#FleetDialog {
    background: #15181f;
}

* {
    outline: 0;
    font: 11px arial;
}

QLabel {
    color: #ffffff;
}



QProgressBar {
    border: none;
    border-radius: 2px;
	text-align:center;
    background: #323947;
	color:#fff;
}

QProgressBar::chunk {
    background-color: #24dba1
}

QPushButton {
    background: #2574ab;
    border-radius: 2px;
	padding: 3px 5px 2px;
    border: none;
    min-width: 100px;
    color: #ffffff;
}

QPushButton::hover {
    background: #2b85c4;
}

QPushButton::pressed {
    background: #206291;
}

QPushButton:disabled {
    background-color: #1c2029;
    color: #46474a;
}
QListView {
    outline: 0;
    color: #ffffff;
    background: transparent;
    border: 1px solid #323947;
	border-radius:2px;
	padding:10px;
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
}

QListView::item {
	padding:10px 12px 9px;
	margin-right:10px;
	border-radius:2px
}

QListView::item:selected {
    color: #fff;
}

QListView::item:selected:!active {
    background: #373f52;
}

QListView::item:selected:active {
    background: #262c38;
}

QListView::item:hover {
    background: #262c38;
}

QScrollBar {
    border: none;
    background: #20252e;
	border-radius:2px;
    width: 5px;
    margin: 0;
}

QScrollBar::handle {
    background: rgba(255,255,255,0.1);
    min-height: 20px;
	border-radius:2px;
}

QScrollBar::handle:hover {
    background: rgba(36,219,161,1);
    min-height: 20px;
	border-radius:2px;
}

QScrollBar::sub-line, QScrollBar::add-line{
    subcontrol-origin: margin;
    height: 20px;
    border: none;
    background: transparent;
	border-radius:2px;
}

QScrollBar::up-arrow, QScrollBar::down-arrowl {
    border: none;
}

QScrollBar::add-page, QScrollBar::sub-page {
    background: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">background:#d9534f;
border-radius:2px;
font-weight:bold;
padding:10px 12px 9px;
color:#fff;</string>
     </property>
     <property name="text">
      <string>Please keep in mind that the larger your fleet is the longer it takes to process. Uploading multiple fleets at once may have the same effect.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="frame_4">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #323947;
border-radius:2px;
padding:10px 12px 9px;
color:#ccc</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rosters that are older than %0 days will not appear.&lt;/p&gt;&lt;p&gt;To obtain a Fleet Roster, open the &lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Fleet Window&lt;/span&gt; navigate to the &lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Roster Tab&lt;/span&gt; and click the &lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Export Button&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="currentRow">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>4</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Processing, please be patient...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>{Spinner Placeholder}</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QFrame" name="frame_5">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="upload_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openBtn">
          <property name="text">
           <string>Open Folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Reload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../nexus.qrc"/>
 </resources>
 <connections/>
</ui>
